### PR TITLE
update portalLib

### PR DESCRIPTION
**Pull request by yuanjunliang**

### CODE REVIEW

File: src/PortalLib.sol [hunk 1]
```diff
@@ -101,11 +101,10 @@ library PortalLib {
...
 
-        uint256 pendingReborn = (portfolio.accumulativeAmount *
-            pool.accRebornPerShare) /
-            PERSHARE_BASE -
-            portfolio.rebornRewardDebt +
-            portfolio.pendingOwnerRebornReward;
+        uint256 pendingReborn = portfolio.pendingOwnerRebornReward +
+            ((portfolio.accumulativeAmount * pool.accRebornPerShare) /
+                PERSHARE_BASE) -
+            portfolio.rebornRewardDebt;
```
Comment (noyyyy) [inline src/PortalLib.sol:107]: ` (portfolio.accumulativeAmount *
            pool.accRebornPerShare) /
            PERSHARE_BASE`  must be larger than `portfolio.rebornRewardDebt`. Or there must be logic error

File: src/PortalLib.sol [hunk 2]
```diff
@@ -151,7 +148,7 @@ library PortalLib {
         portfolio.pendingOwnerNativeReward = 0;
 
         /// @dev send drop
-        if (pendingNative != 0) {
+        if (pendingNative > 0) {
```
Comment (noyyyy) [inline src/PortalLib.sol:151]: I think it's totally the same

File: src/PortalLib.sol [hunk 3]
```diff
@@ -187,20 +184,20 @@ library PortalLib {
 
         // if no portfolio, no pending reward
         if (portfolio.accumulativeAmount == 0) {
-            return (pendingNative, pendingReborn);
+            return (0, 0);
```
Comment (noyyyy) [inline src/PortalLib.sol:187]: store two constants in the contract. it makes size larger

File: src/PortalLib.sol (hunk 2 above)
Comment (yuanjunliang) [reply src/PortalLib.sol:151]: it is helpful for reading code

**Review by noyyyy — APPROVED** [review]

**Pull request merged**